frontend/views/containers/modals/CreateGroup.vue

```diff
@@ -1,48 +1,53 @@
 <template lang='pug'>
-//- TODO: move this into modal
-main.main#create-group-page
-  .steps
-    router-link.step(
+modal-base-template
+  .steps(v-if='currentStep + 1 < config.steps.length')
+    button.step(
       v-for='(step, index) in config.steps'
       :key='index'
-      :to='{name: step}'
       :class='[currentStep === index ? "active" : "", currentStep < index ? "next" : ""]'
+      @click='redirect(step)'
+      v-if='step != "GroupWelcome"'
```

Comment:
Since `"GroupWelcome"` might be renamed, would it make more sense to check against `index` or `currentStep`?

Reply:
Oh, if our eslinter was working this would also complain about `!=` not being `!==`

Reply:
I think it's clearer with a name than an integer actually, it's clearer in the code why it's an exception